src/pages/admin/AdminCustomerInstitutionPage.tsx

```diff
@@ -1,10 +1,143 @@
 import React, { FC } from 'react';
 import Card from '../../components/Card';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import { Field, Form, Formik } from 'formik';
+import * as Yup from 'yup';
+import { Button, TextField } from '@material-ui/core';
+import { CustomerInstitutionFieldNames, emptyCustomerInstitution } from '../../types/customerInstitution.types';
+import Heading from '../../components/Heading';
 
+const StyledFieldWrapper = styled.div`
+  margin: 1rem;
+  flex: 1 0 40%;
+`;
+
+const StyledButtonContainer = styled.div`
+  margin-top: 2rem;
+  margin-right: 1rem;
+  display: flex;
+  justify-content: flex-end;
+`;
 const AdminCustomerInstitutionPage: FC = () => {
+  const { t } = useTranslation('admin');
+
+  const onClickSave = () => {};
+
   return (
     <Card>
-      <form>instituition-form</form>
+      <Heading>{t('add_institution')}</Heading>
+      <Formik
+        initialValues={emptyCustomerInstitution}
+        validationSchema={Yup.object({
+          name: Yup.string().required('Required'),
+        })}
+        onSubmit={(values, { setSubmitting }) => {
+          setTimeout(() => {
+            alert(JSON.stringify(values, null, 2));
+            setSubmitting(false);
+          }, 400);
+        }}>
+        <Form>
+          <StyledFieldWrapper>
```

Comment:
fikk vi ikke til å bruke styled-components direkte på `Field` en gang? eller var det akkurat det vi ikke fikk til?

Reply:
funket det, ja. tenkte bare ikke på det. er for vant å bruke wrappere